cogs/Timezone.py

```diff
@@ -103,9 +116,16 @@ async def get(
         timezone = await self.db_timezone.get_user(user.id)
         if timezone is None:
             return await errorEmb(inter, "This user doesn't have a timezone set")
+
+        # Handles tbilisi naming cause its annoying me.
+        if timezone.timezone == "Asia/Tbilisi":
+            display_timezone = "Europe/Tbilisi"
```

Comment:
can be easily simplified. just do a .replace

Reply:
let me see what i can do